### DISCUSSION TITLE
What are the use cases of the "Custom Stream Link" option?

### DISCUSSION THREAD

**Discussion by distante** [JurajNyiri/HomeAssistant-Tapo-Control#793] [Q&A] [answered]
I have searched the docs and not found any pointer on what does that option does, or the implications of using it.

I am using frigate for recording and restreammig  the camera to prevent too much connections, I also see the cameras en home assistant using Webrtc (from my frigate instance). I do not know if I should set something there for example. O

Thanks for the answers!

**Comment by JurajNyiri** [comment, marked as answer]
If you set this option instead of exposing rtsp stream into HA from camera, integration will use whatever you put in.

This allows you to limit number of streams from camera if you already have camera restreamed via another product.

String has to be in format of for example `rtsp://{username}:{password}@{host}:554/{streamType}`

**Reply by distante** [reply]
Will it also have some effect on some of the sensors?

Thanks!

**Reply by distante** [reply]
@JurajNyiri can we use webrtc streams here?

**Reply by JurajNyiri** [reply]
Whatever URL format HA itself supports should work.
